cypress/integration/unit/publication.spec.js

```diff
@@ -242,5 +242,14 @@ context('Publications tests', () => {
     // Check dossier;
     cy.visit('/publicaties/te-behandelen');
     cy.get(pageClass).should('exist'); // Why not test for the existance of the case (name) ?
+    cy.get('.auk-table__cell--accent').should('exist');
```

Comment:
Kunnen we hier geen gewone selector voor gebruiken?

Reply:
Ik zal dit nog aanpassen, hoeft dit niet tegen te houden